app/controllers/home_controller.rb

```diff
@@ -2,8 +2,19 @@ class HomeController < ApplicationController
 
   def index
     if current_user.present?
-      @activities = PublicActivity::Activity.order("created_at desc").where(owner_id: current_user.id, owner_type: "User")
+      from = Time.now - 90.days
+      to = Time.now
+      tmp = from
+      @array = []
+      begin
+        tmp = tmp + 1.day
+        @array << [
+            day: tmp.strftime("%d %b"),
+            dayago: "#{Date.today.to_date - tmp.to_date} days ago",
```

Comment:
Can we use http://apidock.com/rails/ActionView/Helpers/DateHelper/time_ago_in_words ?